corehq/apps/app_manager/detail_screen.py

```diff
@@ -352,8 +352,39 @@ def variables(self):
         return variables
 
 
+@register_format_type('conditional-enum')
+class ConditionalEnum(Enum):
+    @property
+    def sort_node(self):
+        node = super(ConditionalEnum, self).sort_node
+        if node:
+            node.order = "-2"
+            node.direction = "ascending"
```

Comment:
Sort nodes are not added with variables. This overrides the sort node to add variables for the sort node. Plus adds the order and direction which i found was used for sort nodes as default for example in `tdh.txt` we have `<sort type="string" order="-2" direction="ascending">`